app/components/primer/tooltip_component.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Primer
+  # The Tooltip component is a wrapper component that will apply a tooltip to the provided content.
+  class TooltipComponent < Primer::Component
+    DIRECTION_DEFAULT = :n
+    ALIGN_DEFAULT = :default
+    MULTILINE_DEFAULT = false
+    DELAY_DEFAULT = false
+
+    ALIGN_MAPPING = {
+      ALIGN_DEFAULT => "",
+      :left_1 => "tooltipped-align-left-1",
+      :right_1 => "tooltipped-align-right-1",
+      :left_2 => "tooltipped-align-left-2",
+      :right_2 => "tooltipped-align-right-2",
+    }
+
+    DIRECTION_OPTIONS = [DIRECTION_DEFAULT] + [
+      :nw,
+      :ne,
+      :w,
+      :e,
+      :sw,
+      :s,
+      :se,
+    ]
+
+    # @example 100|Default
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder")) { "Default Bold Text" } %>
+    #   </div>
+    #
+    # @example 100|With a direction
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder", direction: :s)) { "Bold Text With a Direction" } %>
+    #   </div>
+    #
+    # @example 100|With an alignment
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder", direction: :s, alignment: :right_1)) { "Bold Text With an Alignment" } %>
+    #   </div>
+    #
+    # @example 100|Without a delay
+    #   <div class="pt-5">
+    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder", direction: :s, no_delay: true)) { "Bold Text without a delay" } %>
+    #   </div>
+    #
```

Comment:
I'd like to see an example adding tooltips to things other than text 😉 

```suggestion
    #
    # @example 100|Wrapping another component
    #   <div class="pt-5">
    #     <%= render(Primer::TooltipComponent.new(label: "Even bolder")) do %>
    #       <%= render(Primer::ButtonComponent.new) { "Bold Button" } %>
    #     <% end %>
    #   </div>
    #
```

Reply:
Added in 84871ff!